bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -861,37 +861,37 @@ <h3 class="group-title" id="mgmt-foundation"><a href="{{ url('foundation.index')
         <div class="person-bio">
           <p>
           {% trans %}
-            The web is one of our most valuable public resources — it’s Mark Surman’s job to protect it.
+            The internet is one of our most valuable public resources -- it’s Mark Surman’s job to keep it healthy.
           {% endtrans %}
           </p>
 
           <p>
           {% trans %}
-            Mark is Executive Director of the Mozilla Foundation, a global community that does everything from making Firefox to taking stands on issues like privacy and net neutrality. Mark’s main job is to build the movement side of Mozilla, rallying the citizens of the web, building alliances with likeminded organizations and leaders, and growing the open internet movement. Mark’s goal is nothing short of making the health and accessibility of the internet a mainstream issue around the world.
+            Mark is Executive Director of Mozilla Foundation, the global nonprofit that does everything from making Firefox to standing up for issues like online privacy. Mark is focused on fueling the broader internet health movement by working with citizens, technologists and civic leaders around the world. Mark is also part of the team that develops Mozilla’s annual Internet Health Report.
           {% endtrans %}
           </p>
 
           <p>
           {% trans %}
-            Mark has been doing work like this for 20 years: standing up for open source and putting technology into the hands of everyday people. Mark was the founding Director of telecentre.org, a $26M initiative connecting community technology centers in more than 30 countries. He ran the Commons Group for 10 years, a boutique consulting firm that provided advice and insight on networks, technology and social enterprise to nonprofits and governments around the world. Mark was awarded the prestigious Shuttleworth Foundation fellowship, where he explored how to apply open source approaches to philanthropy in the year before he joined Mozilla.
+            Mark has been doing work like this for 20 years: ensuring the internet remains a global public resource, open and accessible to all. Mark was the founding Director of <a rel="external" href="http://www.telecentre.org">telecentre.org</a>, a $26M initiative connecting community technology centers in more than 30 countries. He ran the Commons Group for 10 years, a boutique consulting firm that provided advice and insight on networks, open source and social enterprise to nonprofits and governments around the world. Mark was awarded the prestigious Shuttleworth Fellowship, where he explored how to apply open source approaches to philanthropy in the year before he joined Mozilla.
```

Comment:
`rel="external"` 👍

Reply:
The telecentre.org URL is not resolving for me. The Wayback Machine hasn't been able to crawl it since last year so I think we should remove this link :(